### ISSUE TITLE
branch: refactor-solution-describer

### ISSUE THREAD

**Issue by twant** [phetsims/molarity#73]
Created a new branch to re-do solution describer to remove some of its responsibilites. Still need to: 

- [x] Remove MolarityAlertManager and add those pieces to MolarityScreenView

- [ ] Update focus alert text to match design doc

- [ ] Figure out how to update aria live text without it being read out

- [x] Rename SolutionDescriber to be MolarityDescriber

**Comment by twant** [comment]
Merged into master. Going to delete this branch and continue working on focus alerts on master.

**Comment by zepumph** [comment]
Should be re-deleted as part of phetsims/special-ops#198

**Comment by zepumph** [comment]
Deleted
